### DISCUSSION TITLE
質問のコメントと回答の入力欄の区別を判りやすくして欲しい

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#617] [要望・アイデア・不具合]
![sc1](https://github.com/increments/qiita-discussions/assets/12453432/4103a9a8-ed67-4018-8117-49953816f914)

上は現在のQiitaのコメント入力欄と回答入力欄の境界付近の画面ですが、コメントと回答の左位置が同じで、うっかり回答と回答のコメントの入力を間違える事があります。（私以外にも間違っている方を見かけました）

![sc2](https://github.com/increments/qiita-discussions/assets/12453432/47e677db-1d66-413f-8bdf-ee6d70b644a1)

上はStackoverflowの例ですが、コメントを追加というワンクッションが入ることと、回答欄が他の回答より領域を左に寄せており、また画像では判りませんが、コメントのフォントも回答より小さくなっています。入力箇所を間違えるという事はまず発生しません。

![sc3](https://github.com/increments/qiita-discussions/assets/12453432/a507fa87-8cea-4363-b55d-7dd225b1bc26)

このように、コメント欄を少し小さくして右に寄せるだけでもかなり区別しやすくなると思われますが、何らかのレイアウト改善をお願いしたいです。

**Comment by tanaka-asami415** [comment]
@radian-jp 
ご提案いただきありがとうございます。

現在は、「コメントする」フォームはクリックして広がるような仕様にしているのですが、それでも間違えやすいということでしょうか？🙇
https://github.com/increments/qiita-discussions/discussions/494#discussioncomment-6849605

**Reply by radian-jp** [reply]
自アイコンのX位置が同じため、やはり間違いやすいと思われます。
視覚的にコメント欄である事がわかりやすい方がいいですね。ソースコードもインデントされていると読みやすいですよね。

**Reply by tanaka-asami415** [reply]
@radian-jp 
回答ありがとうございます🙏

時期は未定になりますが、他の方のご意見も参考にしつつ、最初の状態をボタンにする＆少し入力欄の見え方を変更する方向で検討してみます。

アイディアのイメージをデザイナーが作ってくれました！
（※イメージなので、実装時は異なる可能性があります）

| ボタン押下前 | ボタン押下後 |
|:-:|:-:|
| <img width="400" alt="image.png" src="https://github.com/increments/qiita-discussions/assets/103916185/90050368-904e-4505-81e9-fb660b940cee"> |<img width="400" alt="image.png" src="https://github.com/increments/qiita-discussions/assets/103916185/6bf3777f-0667-4db1-9a19-4b28aac9ffc1"> |

**Reply by radian-jp** [reply]
コメント入力欄を表示するのにボタンを押すという操作が必要になるので、判りやすくていいと思います！
